web/routes.go

```diff
@@ -56,3 +56,9 @@ func SysRoute(r *mux.Router) {
 func Metrics(r *mux.Router) {
 	r.Handle("/metrics", prometheus.Handler()).Methods("GET")
 }
+
+// Service template route
```

Comment:
Go comments are suggested to start with the function name they are commenting, like:
```go
// Templates sets up the template route handler for the web interface.
```